### ISSUE TITLE
False positive "no-unused-import": type used in "overrides"

### ISSUE THREAD

**Issue by PaulRBerg** [solhint-community/solhint-community#17]
Getting an "imported name IERC721Metadata is not used" error for the following code:

```solidity

pragma solidity >=0.8.19;

import { ERC721 } from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
import { IERC721Metadata } from "@openzeppelin/contracts/token/ERC721/extensions/IERC721Metadata.sol";

contract MyContract {
    /// @inheritdoc ERC721
    function tokenURI(uint256 streamId) public view override(IERC721Metadata, ERC721) returns (string memory uri) {
        // Checks: the stream NFT exists.
        _requireMinted({ tokenId: streamId });
    
        // Generate the URI describing the stream NFT.
        uri = _nftDescriptor.tokenURI(this, streamId);
    }
}
```

It looks like [overrides](https://docs.soliditylang.org/en/v0.8.19/contracts.html#function-overriding) are not supported.
